.travis.yml

```diff
@@ -1,4 +1,5 @@
 language: node_js
+dist: xenial
```

Comment:
@islemaster do you remember why we made this change?

Reply:
I don't! I did not take good notes in this commit, and it's unclear from the previous test failure why changing distros would fix anything.

If I had to guess, I'd say it's going to be an issue with a native dependency not having a prebuilt package for a particular distro at the time, which might no longer be a problem.

What's default on Travis now?  Do we have a reason not to do this?